__tests__/app.test.js

```diff
@@ -150,29 +151,125 @@ describe("GET /api/articles/:article_id/comments", () => {
       .get("/api/articles/Banana/comments")
       .expect(400)
       .then((res) => {
-        expect(res.error.text).toEqual('bad request')
+        expect(JSON.parse(res.error.text)).toEqual({msg: 'bad request'})
```

Comment:
Why are you parsing here? And a follow up question, why is there an error key on your response? This should be on the body if rejected properly.